README.md

```diff
@@ -80,8 +80,7 @@ Notes:
 
 - In Bash, use **Tab** to complete arguments or list all available commands and **ctrl-r** to search through command history (after pressing, type to search, press **ctrl-r** repeatedly to cycle through more matches, press **Enter** to execute the found command, or hit the right arrow to put the result in the current line to allow editing).
 
-- In Bash, use **ctrl-w** to delete the last word, and **ctrl-u** to delete all the way back to the start of the line. Use **alt-b** and **alt-f** to move by word, **ctrl-a** to move cursor to beginning of line,  **ctrl-e** to move cursor to end of line, **ctrl-k** to kill to the end of the line, **ctrl-l** to clear the screen. See `man readline` for all the default keybindings in Bash. There are a lot. For example **alt-.** cycles through previous arguments, and **alt-*** expands a glob.
-
+- In Bash, use **ctrl-w** to delete the last word, and **ctrl-u** to delete all the way back to the start of the line. Use **alt-b** and **alt-f** to move by word, **ctrl-a** to move cursor to beginning of line,  **ctrl-e** to move cursor to end of line, **ctrl-k** to kill to the end of the line, **ctrl-y** to paste (**alt-y** cycles the kill ring), **ctrl-l** to clear the screen. Use `bind -p` to see the defined keybindings and `man 3 readline` for detailed documentation. There are a lot. For example **alt-.** cycles through previous arguments, **alt-*** expands a glob and **ctrl-alt-e** expands aliases.
```

Comment:
I agree @neil-greenwood :+1: 

I added a link explaining the Bash terminology:

"details about this terminology can be seen in [Bash manual](https://www.gnu.org/software/bash/manual/bash.html#Readline-Killing-Commands)"

What do you think?